app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
```

Comment:
Fetching would need access control, currently it's only possible to fetch public stuff in the protocol. So that's totally out of scope here (and for 0.7.0.0).

We can ask the user which contacts to add again on import, that would fix changed `sharing` states since the backup was created. We can then send new `contact` messages for all contacts from the archive with `sharing`/`following` to `true` or `false`, depending on what the user selected. But that's something for when the import is done, not now.

The only thing we need to do now (for 0.7.0.0) is send `contact` messages again after receiving a `account_migration` for all contacts still `sharing=true` with the migrated user to update the `receiving=true` on the imported users contact. That should be done as part of #6750.